user-guide/Advanced_Functionality/DataMiner_Agents/Restoring_a_DMA/Preparing_the_destination_server_for_a_DMA_restoration.md

```diff
@@ -94,6 +94,22 @@ To prepare the destination server, there are several possibilities:
 
         - *C:\\Program Files\\MySQL\\MySQL Server 5.0\\data\\SLDMADB*
 
+  1. For a DMA using a local Elasticsearch database:
+
+      1. Open Windows Task Manager and stop the Elasticsearch service.
+
+      1. Delete the content of the folder *D:\\ProgramData\\ElasticSearch\\*.
+
+  1. For a DMA using a local OpenSearch database:
+
+      1. Stop OpenSearch by clicking CTRL+C in the command prompt or Powershell. Or simply close the window.
```

Comment:
This step may need a bit more detail. Which command prompt do you mean exactly?

Reply:
I know it's a little bit confusing,

The OpenSearch for Windows jsut runs in an command prompt or Powershell session, OpenSearch has no service that is running. 
From their own information from https://opensearch.org/docs/latest/install-and-configure/install-opensearch/windows/ you can see a note:
To stop OpenSearch, press Ctrl+C in Command Prompt or Powershell, or simply close the Command Prompt or Powershell window.

If still not clear, please let me know.
I tried to make it a little more clear in the guide.